.gitignore

```diff
@@ -8,10 +8,13 @@ coverage
 **/storybook-site
 **/build
 packages/blade/components.js
+packages/blade/components.native.js
```

Comment:
metro older versions doesn't support exports field. Thus we need to let it know that pick things from components.native.js instead. 

Because,
component.js points to `./build/lib/web/production/components`
and component.native.js points to `./build/lib/native/components`

Reply:
do we need the `components.js` now though? because webpack supports the exports field?

Reply:
> because webpack supports the exports field?

We should keep it either way, no harm in it. Best to keep it because we don't know what sort of setup consumers have, having a fallback is a safe option.